sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react"
+import { ListingSearchParams, generateSearchQuery } from "../../../lib/listings/search"
+import { ListingService } from "../../../lib/listings/listing-service"
+import { ListingsCombined } from "../ListingsCombined"
+import { AppearanceBorderType, AppearanceSizeType, Button } from "@bloom-housing/ui-components"
+import { FormOption, ListingsSearchModal } from "./ListingsSearchModal"
+
+type ListingsSearchCombinedProps = {
+  searchString?: string
+  googleMapsApiKey: string
+  listingsEndpoint: string
+  bedrooms: FormOption[]
+  bathrooms: FormOption[]
+  counties: FormOption[]
+}
+
+/**
+ * This combines the search form with the listings map/list. Listings are updated
+ * in both when the search form is submitted.
+ *
+ * @param props
+ * @returns
+ */
+export function ListingsSearchCombined(props: ListingsSearchCombinedProps) {
+  const [listings, setListings] = useState([])
+  const [modalOpen, setModalOpen] = useState(false)
+  const [filterCount, setFilterCount] = useState(0)
+
+  const onFormSubmit = async (params: ListingSearchParams) => {
+    const qb = generateSearchQuery(params)
+    const listingService = new ListingService(props.listingsEndpoint)
+    const result = await listingService.searchListings(qb)
+
+    const listings = result.items
+    const meta = result.meta
+
+    // Keeping this until pagination is implemented
+    console.log(
+      `Showing ${meta.itemCount} listings of ${meta.totalItems} total (page ${meta.currentPage} of ${meta.totalPages})`
```

Comment:
nit: in the log statements this is the same as itemCount, not the total number of items. I see "showing 2 listings of 2 total" for example. Not an issue since it's just a log but pointing it out in case you hadn't seen it yet

Reply:
That's because the query right now is returning all matching listings.  Once pagination is implemented it will only pull a partial list each time